episodes/choosing-acquisition-settings.md

```diff
@@ -0,0 +1,546 @@
+helps us understand how large a region each pixel of our image covers, but (as 
+with magnification) it doesn't tell us the size of details the microscope is 
+really capable of resolving.  For example, I could process a microscopy image 
+to artificially give it 4x as many pixels as the original. This would reduce the 
+pixel size without allowing us to see any smaller details i.e. there's no 
+corresponding improvement in spatial resolution!
+
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Pixel size vs resolution
+
+Consider the 16x16 pixel image below of a circle. If the pixel size in x/y is 
+1.5 micrometre:
+
+- How wide is the circle (in micrometre)?
+- How wide is the entire image (in micrometre)?
+- If the circle is displayed as 9cm wide, then what is the magnification?
+
+
+![](fig/pixel-size-magnification-exercise-1.png){alt="A 16x16 image of a 
+grayscale circle" width="70%"}
+
+Below is a downsampled version of the same image (now 8x8 pixels).
+
+- What is the new pixel size (in micrometre)?
+- How wide is the circle (in micrometre)?
+- How wide is the entire image (in micrometre)?
+- If the circle is displayed as 9cm wide, then what is the magnification?
+
+![](fig/pixel-size-magnification-exercise-2.png){alt="An 8x8 image of a 
+grayscale circle" width="70%"}
+
+:::::::::::::::::::::::: solution 
+
+### Solution
+
+### 16x16 image
+
+The circle is 12 pixels wide, which means it is (12 x 1.5) = 18 micrometre wide
+
+The entire image is 16 pixels wide, which means it is (16 x 1.5) = 24 micrometre 
+wide
+
+9cm is equivalent to 90,000 micrometre. The magnification is therefore 90,000 
+(the displayed size) divided by 18 micrometre (the actual size) = 5000x
+
+### 8x8 image
+
+The image still covers the same total area, but has been downsampled 2x. This 
+means the pixel size will be doubled to 3 micrometre.
+
+As this is an image of the exact same circle as the 16x16 image, the circle 
+width is unchanged. In this image it is 6 pixels wide, which means it is 
+(6 x 3) = 18 micrometre wide (the same as above).
+
+Again, the image still covers the same total area, so its total width will be 
+unchanged too. Here, it's 8 pixels wide, which means it is 
+(8 x 3) = 24 micrometre wide.
+
+The magnification is also unchanged in this case. Magnification depends on the 
+size of an object in the displayed image vs its actual size. If it is still 
+displayed at 9cm wide, then the result will be the same, 5000x.
+
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+### Choosing spatial resolution
+
+Now that we understand the difference between magnification, resolution and 
+pixel size - how do we go about choosing the spatial resolution to use in our 
+experiments? 
+
+The most important thing is to only use the level of resolution that you really 
+require to answer your research question. For example, say you were researching 
```

Comment:
The example you give below is a good one, but i also wonder if this makes it sound as if you should go right up to the edge to what you want to resolve, i.e. you think you only need 5.0um resolution, but perhaps going a little under (3.0,4.0) would hedge your bets a little to make sure you are not under-resolving. Is this a potential concern with microscopy or is it more these big scales of resolution that people are deciding upon?

Reply:
With light microscopy, you tend to be choosing between quite large scales of resolution as you are swapping between physical objective lenses (like 10x, 20x, 40x...).  I've added in some extra information to clarify this + another section on 'nyquist sampling' which controls what the pixel size should be in relation to this.

Reply:
Ah, okay, that is clearer now and makes sense.